### ISSUE TITLE
bug: Configure SSR with Ionic

### ISSUE THREAD

**Issue by iwoogy** [ionic-team/ionic-framework#24073]
### Prerequisites

- [X] I have read the [Contributing Guidelines](https://github.com/ionic-team/ionic-framework/blob/main/.github/CONTRIBUTING.md#creating-an-issue).
- [X] I agree to follow the [Code of Conduct](https://ionicframework.com/code-of-conduct).
- [X] I have searched for [existing issues](https://github.com/ionic-team/ionic-framework/issues) that already report this problem, without success.

### Ionic Framework Version

- [ ] v4.x
- [X] v5.x
- [ ] v6.x

### Current Behavior

Cannot install SSR feature on Ionic

### Expected Behavior

Install SSR on Ionic

### Steps to Reproduce

Documentation followed
https://ionicframework.com/blog/ssr-with-angular-universal-and-ionic/

`nvm use 14.17.1`
Now using node v14.17.1 (npm v7.21.0)

`node_modules/.bin/ng update @angular/core @angular/cli`

<img width="774" alt="image" src="https://user-images.githubusercontent.com/479162/137292162-360fa514-fff7-431b-9b2b-139c8ffcffe9.png">

Let's revert changes to package.json and try to install angular animations

`npm install @angular/animations`

<img width="592" alt="image" src="https://user-images.githubusercontent.com/479162/137292651-335e4e80-7158-45d3-91b6-ae5bc2b9a73b.png">

`node_modules/.bin/ng add @nguniversal/express-engine`

<img width="704" alt="image" src="https://user-images.githubusercontent.com/479162/137292982-3fd59ecc-c806-4e30-a19f-2b7fd5e19bd2.png">

### Code Reproduction URL

https://github.com/iwoogy/ionic-ssr

### Ionic Info

[WARN] Error loading @capacitor/ios package.json: Error: Cannot find module '@capacitor/ios/package'

       Require stack:
       - /Users/dev/.nvm/versions/node/v14.17.1/lib/node_modules/@ionic/cli/lib/project/index.js
       - /Users/dev/.nvm/versions/node/v14.17.1/lib/node_modules/@ionic/cli/lib/index.js
       - /Users/dev/.nvm/versions/node/v14.17.1/lib/node_modules/@ionic/cli/index.js
       - /Users/dev/.nvm/versions/node/v14.17.1/lib/node_modules/@ionic/cli/bin/ionic
[WARN] Error loading @capacitor/android package.json: Error: Cannot find module '@capacitor/android/package'

       Require stack:
       - /Users/dev/.nvm/versions/node/v14.17.1/lib/node_modules/@ionic/cli/lib/project/index.js
       - /Users/dev/.nvm/versions/node/v14.17.1/lib/node_modules/@ionic/cli/lib/index.js
       - /Users/dev/.nvm/versions/node/v14.17.1/lib/node_modules/@ionic/cli/index.js
       - /Users/dev/.nvm/versions/node/v14.17.1/lib/node_modules/@ionic/cli/bin/ionic

Ionic:

   Ionic CLI                     : 6.17.0 (/Users/dev/.nvm/versions/node/v14.17.1/lib/node_modules/@ionic/cli)
   Ionic Framework               : @ionic/angular 5.8.4
   @angular-devkit/build-angular : 12.1.4
   @angular-devkit/schematics    : 12.2.10
   @angular/cli                  : 12.1.4
   @ionic/angular-toolkit        : 4.0.0

Capacitor:

   Capacitor CLI      : 3.2.5
   @capacitor/android : not installed
   @capacitor/core    : 3.2.5
   @capacitor/ios     : not installed

Utility:

   cordova-res : 0.15.3
   native-run  : 1.5.0

System:

   NodeJS : v14.17.1 (/Users/dev/.nvm/versions/node/v14.17.1/bin/node)
   npm    : 7.21.0
   OS     : macOS Big Sur

**Comment by iwoogy** [comment]
Hello

Sorry my bad. Repo is now public.

Regards

Bobby

**Comment by liamdebeasi** [comment]
Thanks! I am not able to reproduce this on my end. I tested this using npm 7.21.0 and `ng update` ran successfully. 

Looking at this closer, this appears to be related to either your environment setup or your project configuration, not Ionic Angular (or it could be a defect in npm 7). I did a quick search on the Angular CLI repo, and it looks like other users have run into similar issues: https://github.com/angular/angular-cli/issues/21204. The recommendation there was to pass the `--force` option to `ng update`.

I am going to close this as there is nothing actionable from the Ionic end. If you run into additional issues I recommend creating a post on the [Ionic Forum](https://forum.ionicframework.com/).
